loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
It's a kind of pattern that many django apps reuse, it's like being uneasy about calling something "models" :) I can make the change, but it doesn't really cause any harm since django.conf.settings is explicitly renamed above.